### ISSUE TITLE
Pass result to callback?

### ISSUE THREAD

**Issue by JasonSanford** [AppGeo/node-arcgis-featureservice#1]
Hi. I'm using this module to create and update AGS features and for my use case I needed the objectid/globalid returned when a feature is added. These are returned in the results from AGS so I've modified my fork to include the result.

https://github.com/JasonSanford/node-arcgis-featureservice/commit/43fbf22e18f01bc55c9da66fea69d8b51ac62233

If this is something you'd consider adding to this module I'll get a proper PR together with some tests. If not feel free to close.

Thanks!
Jason

**Comment by knownasilya** [comment]
I think we would. :+1:

**Comment by ismyrnow** [comment]
Agreed. That sounds like a good enhancement, especially if it aligns with how AGS works, and what users would expect.
